.build/dependency-check-suppressions.xml

```diff
@@ -20,13 +20,6 @@
   https://jeremylong.github.io/DependencyCheck/general/suppression.html
 -->
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-    <suppress>
```

Comment:
The build completes successfully without commented cve suppressions, do we still need them?

```
    <suppress>
        <!--  https://issues.apache.org/jira/browse/CASSANDRA-16150 -->
        <packageUrl regex="true">^pkg:maven/org\.yaml/snakeyaml@.*$</packageUrl>
<!--        <cve>CVE-2023-2251</cve>-->
        <cve>CVE-2022-38752</cve>
        <cve>CVE-2022-38751</cve>
        <cve>CVE-2022-38750</cve>
        <cve>CVE-2022-41854</cve>
        <cve>CVE-2022-25857</cve>
        <cve>CVE-2022-38749</cve>
        <cve>CVE-2022-1471</cve>
<!--        <cve>CVE-2022-3064</cve>-->
<!--        <cve>CVE-2021-4235</cve>-->
        <cve>CVE-2017-18640</cve>
    </suppress>
    
        <!-- https://issues.apache.org/jira/browse/CASSANDRA-15417 -->
    <suppress>
        <packageUrl regex="true">^pkg:maven/io\.netty/netty\-all@.*$</packageUrl>
        <cve>CVE-2019-16869</cve>
        <cve>CVE-2019-20444</cve>
        <cve>CVE-2019-20445</cve>
        <cve>CVE-2020-7238</cve>
        <cve>CVE-2021-21290</cve>
        <cve>CVE-2021-21295</cve>
        <cve>CVE-2021-21409</cve>
        <cve>CVE-2021-37136</cve>
        <cve>CVE-2021-37137</cve>
        <cve>CVE-2021-43797</cve>
        <cve>CVE-2022-24823</cve>
        <cve>CVE-2022-41881</cve>
<!--        <cve>CVE-2022-41915</cve>-->
        <cve>CVE-2023-34462</cve>
        <cve>CVE-2023-44487</cve>
    </suppress>
    
    <suppress>
        <packageUrl regex="true">^pkg:maven/com\.thinkaurelius\.thrift/thrift-server@.*$</packageUrl>
        <cve>CVE-2015-3254</cve>
        <cve>CVE-2016-5397</cve>
<!--        <cve>CVE-2018-1320</cve>-->
<!--        <cve>CVE-2018-11798</cve>-->
        <cve>CVE-2019-0205</cve>
    </suppress>
    
        <suppress>
        <packageUrl regex="true">^pkg:maven/org\.codehaus\.jackson/jackson\-mapper\-asl@.*$</packageUrl>
        <cve>CVE-2017-7525</cve>
<!--        <cve>CVE-2017-15095</cve>-->
<!--        <cve>CVE-2017-17485</cve>-->
<!--        <cve>CVE-2018-5968</cve>-->
<!--        <cve>CVE-2018-14718</cve>-->
<!--        <cve>CVE-2018-1000873</cve>-->
<!--        <cve>CVE-2018-7489</cve>-->
        <cve>CVE-2019-10172</cve>
<!--        <cve>CVE-2019-14540</cve>-->
<!--        <cve>CVE-2019-14893</cve>-->
<!--        <cve>CVE-2019-16335</cve>-->
<!--        <cve>CVE-2019-17267</cve>-->
    </suppress>
```

Reply:
how did you found that? I expected the plugin to output those exclusions which are stale but it did not - if you spend time to manually identify those, there is no reason to not apply that :)

Reply:
I've checked all of them locally :-) nice that we can reduce the changes